dnf-behave-tests/features/steps/common/file.py

```diff
@@ -44,3 +94,51 @@ def read_file_contents(filename, encoding="utf-8"):
 def copy_tree(source, destination):
     shutil.copytree(source, destination)
     assert os.path.exists(destination), "copy_tree {} -> {} failed".format(source, destination)
+
+def copy_file(source, destination):
+    shutil.copyfile(source, destination)
```

Comment:
shutil.copyfile would probably raise Error in case of troubles with copying (before your assert checks it on the next line). May be just wrap it in try: block.